torch/_inductor/codecache.py

```diff
@@ -703,10 +824,11 @@ def _load_library(path):
     @classmethod
     def load(cls, source_code):
         picked_vec_isa = pick_vec_isa()
+        cpp_command = repr(cpp_compile_command("i", "o", vec_isa=picked_vec_isa))
         key, input_path = write(
             source_code,
+            code_hash(source_code + cpp_command),
```

Comment:
Nitty nit, but for robustness you probably need some real delimiter here

Reply:
this is what the code was doing previously, but I'll update with a delimiter